viper.py

```diff
@@ -11,6 +11,12 @@
 cve_list_output_json_file = 'cve_list.json'
 output_jsonl_file = 'data/vidata.json'
 
+if os.environ.get('API') is None:
+  API = 'api.kennasecurity.com'
+else:
+  API =  os.environ.get('API')
```

Comment:
I'm not sure what kind of validation we can or want to do here, but we'll presumably get weird errors later on if someone specifies something like `https://api.eu.kennasecurity.com`  Ensuring that there's not a protocol prefix would be helpful, maybe we also want to enforce that it ends in `kennasecurity.com`?

Reply:
This is just a quick patch to fix it for a user.  I changed the var to `API_Host` but will leave the error checking until a Sunday afternoon when I am bored.